cmd/serve/serve.go

```diff
@@ -224,24 +224,13 @@ func NewCommand(
 			inventoryresources_repo := inventoryResourcesRepo.New(db)
 
 			//v1beta2
-			// wire together resource handling
+
+			// wire together inventory service handling
 			resource_repo := resourcerepo.New(db)
-			resource_controller := resourcesctl.New(resource_repo, inventoryresources_repo, authorizer, eventingManager, "notifications", log.With(logger, "subsystem", "notificationsintegrations_controller"), storageConfig.Options.DisablePersistence, listenManager, consistencyConfig.ReadAfterWriteEnabled, consistencyConfig.ReadAfterWriteAllowlist)
-			resource_service := resourcesvc.NewKesselResourceServiceV1beta2(resource_controller)
-			pbv1beta2.RegisterKesselResourceServiceServer(server.GrpcServer, resource_service)
-			pbv1beta2.RegisterKesselResourceServiceHTTPServer(server.HttpServer, resource_service)
-
-			// wire together check service
-			check_controller := resourcesctl.New(resource_repo, inventoryresources_repo, authorizer, eventingManager, "authz", log.With(logger, "subsystem", "authz_controller"), storageConfig.Options.DisablePersistence, listenManager, consistencyConfig.ReadAfterWriteEnabled, consistencyConfig.ReadAfterWriteAllowlist)
-			check_service := resourcesvc.NewKesselCheckServiceV1beta2(check_controller)
-			pbv1beta2.RegisterKesselCheckServiceServer(server.GrpcServer, check_service)
-			pbv1beta2.RegisterKesselCheckServiceHTTPServer(server.HttpServer, check_service)
-
-			// wire together lookup service
-			streamedlist_controller := resourcesctl.New(resource_repo, inventoryresources_repo, authorizer, eventingManager, "authz", log.With(logger, "subsystem", "authz_controller"), storageConfig.Options.DisablePersistence, listenManager, consistencyConfig.ReadAfterWriteEnabled, consistencyConfig.ReadAfterWriteAllowlist)
-			streamedlist_service := resourcesvc.NewKesselLookupServiceV1beta2(streamedlist_controller)
-			pbv1beta2.RegisterKesselStreamedListServiceServer(server.GrpcServer, streamedlist_service)
-			//TODO: http service not getting generated
+			inventory_controller := resourcesctl.New(resource_repo, inventoryresources_repo, authorizer, eventingManager, "notifications", log.With(logger, "subsystem", "notificationsintegrations_controller"), storageConfig.Options.DisablePersistence, listenManager, consistencyConfig.ReadAfterWriteEnabled, consistencyConfig.ReadAfterWriteAllowlist)
```

Comment:
More of an FYI that _shouldn't_ be an issue. But since we are updating the namespace to notifications this will nullify this MR https://github.com/project-kessel/inventory-api/pull/537. I believe this will have no impact because `notifications` is the value we are expecting.

I'm looking at a long-term fix for tomorrow.

Reply:
In general, I think this value should come from customer input or config.

Reply:
100% we are handling namespaces a little inconsistently today (mixed between using `uc.Namespace` and also from the customer request)